### PR TITLE
Mission planner bt

### PR DESCRIPTION
**Pull request by mhpanah**
Created a dummy Mission Planner shell. This is based on Behavior Tree library from:
https://github.com/miccol/Behavior-Tree 
When ActionLib is ready, we can convert it to ROS2.

### CODE REVIEW

File: src/MissionPlanner_BT/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
+
+find_package(BTpp REQUIRED)
```
Comment (orduno) [inline src/MissionPlanner_BT/CMakeLists.txt]: should BTpp be BTcpp?

File: src/MissionPlanner_BT/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Mission Planner Behaivor Tree
```
Comment (orduno) [inline src/MissionPlanner_BT/README.md]: Behaivor -> Behaviour

**Comment by crdelsey** [comment]
This change brings in the ros2_devel branch. I don't think we want that in master yet.

**Comment by mjeronimo** [comment]
Mohammad, we can close this PR as I've got the behavior tree functionality in the other pull request.

**Comment by mhpanah** [comment]
Example on how to use BT library.

**Pull request closed without merging**